docs/UserGuide.md

```diff
@@ -321,8 +358,11 @@ Clientele+ data is saved automatically as a JSON file `[JAR file location]/data/
 Data of archived clients is saved automatically as a separate JSON file `[JAR file location]/data/archivedAddressBook.json`.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+
+<br>
 If your changes to the data file makes its format invalid, Clientele+ will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
 Furthermore, certain edits can cause the Clientele+ to behave in unexpected ways (e.g., if a value entered is outside of the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
+
 </div>
 
 ### Archiving data files `[coming in v2.0]`
```

Comment:
By any chance, could we remove this section? It feels too similar to the archive command Jun Kiat implemented. Or perhaps instead of deleting, we could replace it with unarchive?

Reply:
You are right, I will replace it with unarchive